bumpversion/cli.py

```diff
@@ -455,6 +460,13 @@ def _parse_arguments_phase_3(remaining_argv, positionals, defaults, parser2):
         help="Version that needs to be updated",
         required="current_version" not in defaults,
     )
+    parser3.add_argument(
+        "--no-configured-files",
+        action="store_true",
+        default=False,
+        dest="no_configured_files",
+        help="Don't replace the version in the configured files.",
```

Comment:
Maybe it should specify what it does instead of what it doesn't. Not sure the phrasing is really the best, but something like:
```suggestion
        help="Only replace replace the version in files specified on the command line, ignoring the files from the configuration file.",
```

Reply:
Yes, I like this much more